conf/distro/nilrt.conf

```diff
@@ -2,9 +2,9 @@ require nilrt.inc
 
 DISTRO_NAME = "NI Linux Real-Time"
 
-DISTRO_VERSION = "8.7"
+DISTRO_VERSION = "8.8"
```

Comment:
Just a sanity check, this branch (sumo) is always going to be bumping only minor versions, correct?

Reply:
Unless someone convinces me otherwise; yes.